Alignment/MillePedeAlignmentAlgorithm/test/deadmodules.txt

```diff
@@ -0,0 +1,17 @@
+process.TrackerTrackHitFilter.detsToIgnore = [
+      # see https://hypernews.cern.ch/HyperNews/CMS/get/tracker-performance/484.html
+    # TIB / TID
+    #369136710, 369136714, 402668822,
+    # TOB
+    #436310989, 436310990, 436299301, 436299302,
+    # TEC
+    #470340521, 470063045, 470063046, 470114669, 470114670, 470161093, 470161094, 470164333, 470164334, 470312005, 470312006, 470312009, 470067405, 470067406, 470128813
+344070404, #outlier: large rotation around gamma (0.2 rad)
+402672394, 
+352459012,
+344070404,
+369125398,
+369141862,
+470116592,
+
```

Comment:
the reference points to a 2008 HN. Are your really still using that bad module list?